frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -1139,11 +1146,37 @@ export async function fetchMarkerGenes({
 
   return json;
 }
+
+export async function fetchGeneInfo({
+  geneID,
+  geneSymbol,
+  signal,
+}: FetchGeneInfoParams): Promise<GeneInfo> {
+  const url =
+    API_URL + API.WMG_GENE_INFO + `?geneID=${geneID}&gene=${geneSymbol}`;
+  const response = await fetch(url, {
+    ...DEFAULT_FETCH_OPTIONS,
```

Comment:
needed for CORS

Reply:
this might be useful information in the code